llvm/test/MC/AArch64/patchinst.s

```diff
@@ -0,0 +1,7 @@
+// RUN: llvm-mc -triple aarch64-elf -filetype=obj %s -o - | llvm-objdump -r - | FileCheck %s
+
+// Test that PATCHINST appears after JUMP26.
+// CHECK:      R_AARCH64_JUMP26
+// CHECK-NEXT: R_AARCH64_PATCHINST
+.reloc ., R_AARCH64_PATCHINST, ds
+b f1
```

Comment:
Improve the test to test that with more than one fragments and more than one PATCHINST, the relocations are still ordered.
```
.reloc ., R_AARCH64_PATCHINST, ds
b f1
.balign 8
.reloc ., R_AARCH64_PATCHINST, ds
b f2
```

Reply:
Done in 55e71c0b145b40f48f882bd439a6fa99b761d541